collector/disk_usage.go

```diff
@@ -0,0 +1,109 @@
+package collector
+
+import (
+	"context"
+
+	pgx "github.com/jackc/pgx/v4"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+// Queries taken from https://wiki.postgresql.org/wiki/Disk_Usage
+const (
+	relationUsageQuery = `
+	SELECT
+			nspname
+		, relname
+		, pg_relation_size(C.oid) AS "size_bytes"
+  FROM pg_class C
+  LEFT JOIN pg_namespace N ON (N.oid = C.relnamespace)
+  WHERE nspname NOT IN ('pg_catalog', 'information_schema')
+  ORDER BY pg_relation_size(C.oid) DESC /*postgres_exporter*/`
+
+	tableUsageQuery = `
```

Comment:
@isaacseymour as we discuss this would rewrite this way:

```
SELECT schemaname
     , relname
    , pg_table_size(schemaname || '.' || relname)::float
  FROM pg_stat_user_tables /*postgres_exporter*/`
```

This will give us only the table size, including the toast tables and other things, but without the indices size. To get the whole size we can do an aggregation on Prometheus via a recording rule